lib/datatrans/json/transaction/settle.rb

```diff
@@ -0,0 +1,41 @@
+require 'httparty'
+require 'datatrans/json/transaction/response'
+
+class Datatrans::JSON::Transaction
+  class Settle
+    # Class to settle a transaction https://api-reference.datatrans.ch/#tag/v1transactions/operation/settle
+    attr_accessor :params, :datatrans
+
+    def initialize(datatrans, params)
+      @datatrans = datatrans
+      @params = params
+    end
+
+    def post(url, options = {})
+      options = options
+        .merge(self.datatrans.proxy)
+        .merge(:basic_auth => { :username => self.datatrans.merchant_id, :password => self.datatrans.password })
```

Comment:
also a separate PR, but introducing `standardrb` would be nice to get rid of these pre-1.9 hash styles.